src/openrct2/entity/Yaw.hpp

```diff
@@ -13,10 +13,21 @@
 
 namespace OpenRCT2::Entity::Yaw
 {
+    enum class SpritePrecision : uint8_t
+    {
+        Sprites0 = 0,
+        Sprites1,
+        Sprites2,
+        Sprites4,
+        Sprites8,
+        Sprites16,
+        Sprites32,
+        Sprites64
+    };
+
     constexpr const int32_t BaseRotation = 32;
 
-    // smallest precision is 4 frames
-    constexpr const uint8_t PrecisionOffset[] = { 3, 2, 1, 0, 0, 0, 0, 0 };
+    constexpr const uint8_t PrecisionOffset[] = { 5, 5, 4, 3, 2, 1, 0, 0 };
```

Comment:
Perhaps add a comment that the duplicate 5 is for the null/none state

Reply:
And add a comment that 64 frames not currently supported. Base rotation should also have a comment about it being tied to sprites 32